wiki/Gameplay/en.md

```diff
@@ -20,6 +20,7 @@
 - [Mind block](/wiki/Gameplay/Mind_block)
 - [Notelock](/wiki/Gameplay/Judgement/Notelock)
 - [Pass](/wiki/Gameplay/Pass)
+- [Play Style](/wiki/Gameplay/Play_style)    
```

Comment:
the title of the article in question isn't in title case

```suggestion
- [Play style](/wiki/Gameplay/Play_style)    
```

Reply:
Will change, should I also change Play Time to Play time then since I did it because it was capitalized there as well and it seems to be the only instance of capitalization